pkg/tarfs/parse.go

```diff
@@ -185,7 +185,7 @@ func parseNumber(b []byte) (int64, error) {
 	if len(b) == 0 {
 		return 0, nil
 	}
-	n, err := strconv.ParseUint(cstring(b), 8, 64)
+	n, err := strconv.ParseUint(cstring(b), 8, 63) // Only positive int64 values allowed.
```

Comment:
What is the point in Parsing into a `uint64` and then returning a `int64` here? A quick way to check for a negative number?

Reply:
The `strconv.ParseInt` function allows for a `+` or `-`, which `strconv.ParseUint` doesn't. The return type being an `int64` is dictated by other parts of the API, it just happens that they cause friction here.